### ISSUE TITLE
quantum_operator.expt_value broken.

### ISSUE THREAD

**Issue by weinbe58** [weinbe58/QuSpin#248]
quantum_operator.expt_value is throwing an error:

```
quantum_operator_core.py", line 638, in expt_value
    if ishamiltonian(V):
NameError: name 'ishamiltonian' is not defined
```

**Comment by mgbukov** [comment]
@weinbe58 can u upload a simple script that reproduces the bug. I can try to fix it.

**Comment by weinbe58** [comment]
fixed in #205
